.idea/aws.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That's a redundant config file, consider to drop it.

Reply:
Done